google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -492,6 +494,48 @@ public Query whereEqualTo(@Nonnull FieldPath fieldPath, @Nullable Object value)
     }
   }
 
+  /**
+   * Creates and returns a new Query with the additional filter that documents must contain the
+   * specified field and the value does not equal the specified value.
```

Comment:
This might be better (but no need to change unless we update all of the documentation):

```suggestion
   * specified field and its value does not equal the specified value.
```

Reply:
Note that this and the next comment apply to multiple places in this file.